### PR TITLE
chore(release): v3.13.3

### PR DESCRIPTION
**Pull request by Alex-Werner**
## Issue being fixed or feature implemented

This PR update wallet-lib to 7.13.2 which brings multiples fixes and an additional features. 
Especially it adds fix for identity listing and therefore fix a usage issue on registering documents.
As well as an initial sync up fix so we do ensure to sync ALL addresses immediately before releasing (which in the previous build were done later during the app execution). 

## What was done?

- chore: update wallet-lib to 7.13.2

## Checklist:

- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have added or updated relevant unit/integration/functional/e2e tests
- [X] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by Alex-Werner** [comment]
Additional fix incoming, adding new tags in the meantime.

**Comment by Alex-Werner** [comment]
Rebased on top of master

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,37 +1,35 @@
...
-
-### Bug Fixes
-
-* more than one identity registration failed ([#83](https://github.com/dashevo/DashJS/issues/83))
+## [3.13.3](https://github.com/dashevo/DashJS/compare/v3.13.2...v3.13.3) (2020-06-15)
 
+- **Features:**
+    * Updated wallet-lib to [7.13.3](https://github.com/dashevo/wallet-lib/blob/master/CHANGELOG.md#7133-2020-06-16)
+    * Updated js-dpp to [0.13.1](https://github.com/dashevo/js-dpp/blob/master/CHANGELOG.md#0131-2020-06-15)
+
```
Comment (shumkov) [inline CHANGELOG.md:6]: There are two more fixes from Anton. Please check master.

Comment (Alex-Werner) [reply CHANGELOG.md]: Good catch !

**Pull request merged**